examples/proxy/record-encryption/cluster-ip/README.md

```diff
@@ -1,54 +1,63 @@
 # Streams for Apache Kafka Proxy Record Encryption, exposed using Cluster-IP
 
-In this example, an instance of Apache Kafka is deployed using AMQ Stream.  The instance is proxied using
-Streams for Apache Kafka Proxy configured with Record Encyption.  The proxy is exposed within the cluster using a
+In this example, an instance of Apache Kafka is deployed using Streams for Apache Kafka.  The instance is proxied using
+Streams for Apache Kafka Proxy is configured with Record Encryption.  The proxy is exposed within the cluster using a
 Cluster-IP.
 
-# Prerequsistes
+# Prerequisites
 
 * [KMS is prepared](../PREPARE_KMS.md).
-* Vault CLI
+* Vault CLI or AWS CLI 
 
-* Administrative access to the OpenShift Cluster being used to evaluate AMQ Stream Proxy
+* Administrative access to the OpenShift Cluster being used to evaluate Streams for Apache Kafka Proxy
 * OpenShift CLI (oc)
-* AMQ Stream Operator installed namespace wide
+* Streams for Apache Kafka Operator (installed namespace wide)
 
 # Deploying the Example
 
-1. Deploy the Example
+1. Edit `cluster-ip/proxy/proxy-config.yaml` and `base/proxy/kustomization.yaml`. Uncomment either the
+   configuration for Vault or AWS, depending on your KMS provider:
+
+2. Deploy the Example:
    ```sh
    oc apply -k cluster-ip
    ```
 
 # Try out the example
 
-1. Create a key for topic `trades` in the KMS:
-   ```
+1. Create a key for topic `trades` using the instructions applicable to your KMS provider:
+
+   Vault:
+   ```sh
    vault write -f transit/keys/KEK_trades
    ```
-2. Create a topic `trades` on the cluster, via the proxy:
+   AWS:
+   ```sh
+   aws kms create-alias --alias-name alias/KEK_trades --target-key-id $(aws kms create-key | jq -r '.KeyMetadata.KeyId')
    ```
-   oc run -n proxy -qi proxy-producer --image=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0 --rm=true --restart=Never -- bin/kafka-topics.sh --bootstrap-server proxy-service:9092 --create -topic trades
+2. Create a topic `trades` on the cluster, via the proxy:
+   ```sh
+   oc run -n kafka-proxy -qi create-topic --image=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0 --rm=true --restart=Never -- bin/kafka-topics.sh --bootstrap-server proxy-service:9092 --create -topic trades
```

Comment:
it should be `--topic` parameter, right?

Reply:
yes.